config/core/test-configs.yaml

```diff
@@ -2171,6 +2177,24 @@ test_configs:
       - baseline
       - baseline-nfs
 
+  - device_type: juno-uboot
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - kselftest-cpufreq
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-lkdtm
+      - kselftest-rtc
+      - kselftest-seccomp
+      - ltp-crypto
+      - ltp-fcntl-locktests
+      - ltp-ipc
+      # ltp-mm - runs system out of memory
+      - ltp-pty
+      - ltp-timers
+      - smc
```

Comment:
The test wasn't actually run due to a known issue in test-definitions:
https://lava.sirena.org.uk/scheduler/job/84922

So this is "OK" as far as this PR is concerned, although we don't know if it would have produced useful results. I think we can leave it here and review things later when the smc test has been fixed.